### PR TITLE
Use Doctoc to maintain README "Table of contents"!

### PR DESCRIPTION
**Pull request by PeterDaveHello**
This will help maintain "Table of contents" in README very easily!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,8 @@
     "test-ci": "taper tests/test-*.js",
     "test-cov": "istanbul cover tape tests/test-*.js",
     "test-browser": "node tests/browser/start.js",
-    "lint": "standard"
+    "lint": "standard",
+    "doctoc": "doctoc --title='## Table of contents' --github README.md"
```
Comment (RichardLitt) [inline package.json]: Could be changed to `Table of Contents`

Comment (PeterDaveHello) [reply package.json]: Updated, thanks.

**Comment by PeterDaveHello** [comment]
@simov @mikeal would you like to take a look? Thanks :)

File: package.json (hunk 1 above)
Comment (PeterDaveHello) [reply package.json]: Just a reminder, `contents` is currently being used in the README.md

**Comment by PeterDaveHello** [comment]
@reconbot would you like to take a look? Thanks :)

**Pull request closed without merging**